src/App.css

```diff
@@ -106,6 +107,54 @@
   background: lightgray;
 }
 
+#dialog {
```

Comment:
did you add this CSS in both PRs?

Reply:
Sorry, they should be put in the same pull request